consensus/istanbul/validator/weighted_random_test.go

```diff
@@ -441,32 +443,92 @@ func TestWeightedCouncil_SetSubGroupSize(t *testing.T) {
 }
 
 func TestWeightedCouncil_SubListWithProposer(t *testing.T) {
-	validators := makeTestValidators(testNonZeroWeights)
-	valSet := makeTestWeightedCouncil(testNonZeroWeights)
+	var (
+		validators = makeTestValidators(testNonZeroWeights)
+		prevHash   = crypto.Keccak256Hash([]byte("This is a test"))
+		valSet     = makeTestWeightedCouncil(testNonZeroWeights)
+
+		expectIndexOfSubsetLenTest                     = []int{1, 2, 7, 3, 11, 6, 9, 4, 0, 8, 12, 10}
+		expectIndexOfRoundTestBeforeIstanbulCompatible = [][]int{
```

Comment:
It looks fine, but it would be more intuitive if arrays are sorted.

Reply:
I think sorting cannot be done..
In SubsetLen test, the test is organized in a structure where valSet is increased by 1. If subsetLen is 2, expectIndexOfRoundTestBeforeIstanbulCompatible[0][0]-[0][1] is used, and if subsetLen is 8, expectIndexOfRoundTestBeforeIstanbulCompatible[0][0]-[0][7] is used. So I think sorting seems to be difficult.